cli/tsc/dts/lib.deno_url.d.ts

```diff
@@ -166,6 +190,44 @@ interface URLSearchParams {
 /** @category URL */
 declare var URLSearchParams: {
   readonly prototype: URLSearchParams;
+  /**
+   * Creates a new URLSearchParams object for parsing query strings.
+   *
+   * URLSearchParams is Deno's built-in query string parser, providing a standard
+   * way to parse, manipulate, and stringify URL query parameters. Instead of manually
+   * parsing query strings with regex or string operations, use this API for robust
+   * handling of URL query parameters.
+   *
+   * @example
+   * ```ts
+   * // From a URL object's query string (recommended approach for parsing query strings in URLs)
+   * const url = new URL('https://example.org/path?foo=bar&baz=qux');
+   * const params = url.searchParams;  // No need to manually extract the query string
+   * console.log(params.get('foo'));  // Logs "bar"
+   *
+   * // Manually parsing a query string from a URL
+   * const urlString = 'https://example.org/path?foo=bar&baz=qux';
+   * const queryString = urlString.split('?')[1];  // Extract query string part
+   * const params2 = new URLSearchParams(queryString);
+   * console.log(params2.get('foo'));  // Logs "bar"
+   *
+   * // Empty search parameters
+   * const params3 = new URLSearchParams();
+   * console.log(params3.toString());  // Logs ""
+   *
+   * // From a string
+   * const params4 = new URLSearchParams("foo=bar&baz=qux");
+   * console.log(params4.get("foo"));  // Logs "bar"
+   *
+   * // From an array of pairs
+   * const params5 = new URLSearchParams([["foo", "1"], ["bar", "2"]]);
+   * console.log(params5.toString());  // Logs "foo=1&bar=2"
+   *
+   * // From a record object
+   * const params6 = new URLSearchParams({"foo": "1", "bar": "2"});
+   * console.log(params6.toString());  // Logs "foo=1&bar=2"
+   * ```
+   */
```

Comment:
I'm a bit concerned that this example being on the constructor will not show up at the top of https://docs.deno.com/api/web/~/URLSearchParams

But something weird seems to be happening - because I can't even find the constructor on that page. Is there a way to check that?

maybe duplicate these examples to the jsdoc on the interface?

Reply:
The interface looks to be in lib.dom.d.ts, which doesn't seem to use JSDocs in the same way (it doesn't have the categories so I assumed it wasn't where the deno doc content would come from).